drm/radeon: add Mullins chip family

Mullins is a new CI-based APU.

Signed-off-by: Samuel Li <samuel.li@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
diff -Naur a/drivers/gpu/drm/radeon/radeon_asic.c b/drivers/gpu/drm/radeon/radeon_asic.c
--- a/drivers/gpu/drm/radeon/radeon_asic.c	2014-04-26 01:51:28.161070642 +0530
+++ b/drivers/gpu/drm/radeon/radeon_asic.c	2014-04-26 01:58:12.613084276 +0530
@@ -2484,6 +2484,7 @@
 		break;
 	case CHIP_KAVERI:
 	case CHIP_KABINI:
+	case CHIP_MULLINS:
 		rdev->asic = &kv_asic;
 		/* set num crtcs */
 		if (rdev->family == CHIP_KAVERI) {
diff -Naur a/drivers/gpu/drm/radeon/radeon_device.c b/drivers/gpu/drm/radeon/radeon_device.c
--- a/drivers/gpu/drm/radeon/radeon_device.c	2013-11-04 05:11:51.000000000 +0530
+++ b/drivers/gpu/drm/radeon/radeon_device.c	2014-04-26 01:59:11.645085768 +0530
@@ -98,6 +98,7 @@
 	"BONAIRE",
 	"KAVERI",
 	"KABINI",
+	"MULLINS",
 	"LAST",
 };
 
diff -Naur a/drivers/gpu/drm/radeon/radeon_family.h b/drivers/gpu/drm/radeon/radeon_family.h
--- a/drivers/gpu/drm/radeon/radeon_family.h	2013-11-04 05:11:51.000000000 +0530
+++ b/drivers/gpu/drm/radeon/radeon_family.h	2014-04-26 01:59:38.813087463 +0530
@@ -96,6 +96,7 @@
 	CHIP_BONAIRE,
 	CHIP_KAVERI,
 	CHIP_KABINI,
+	CHIP_MULLINS,
 	CHIP_LAST,
 };
 
